### ISSUE TITLE
ConTeXt: startplacetable "caption" vs "title"

### ISSUE THREAD

**Issue by aksdb** [jgm/pandoc#4289]
Pandoc Version: 2.1.1
Commandline: pandoc -t context -s test.md -o test.tex && context test.tex

```
+-----+-----+
| Foo | Bar |
+-----+-----+

Table: Some Caption
```

Result: no caption visible in PDF
Expected: a caption under the table.

Reason:
In the generated TeX it reads `\startplacetable[caption={Some Caption}]` while it should be `\startplacetable[title={Some Caption}]`.

**Comment by mb21** [comment]
@hmenke was this introduced with https://github.com/jgm/pandoc/pull/4223?

**Comment by hmenke** [comment]
@mb21 You are correct and evidently I am an idiot.  Fixed in #4291.
